javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.gyb

```diff
@@ -0,0 +1,630 @@
+%{
+## ArraysJDK9Methods.scala.gyb
+##
+##
+## To generate this file's output manually, execute the python script
+## 'scripts/gyb.py' under the project root. For example, from the project root:
+##
+##   scripts/gyb.py \
+##     javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.gyb \
+##     --line-directive '' \
+##     -o  javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.template
+##
+## Then manually fold the desired parts into Arrays.scala.
```

Comment:
The problem with manual folding is the need of manual labor. Instead, we can make the template generate 
```
private[util] trait ArrayJDK9Methods { self: Arrays.type =>  
...
}
```
which would later be extended by `object Arrays` 

I believe it should be binary compatible from the perspective of NIR when it comes to symbols expected from the JDK we're compiling against

Reply:
The excellent & implemented suggestion show the difference between the work & thinking of
a black belt and that of a green belt.  Thank you.

It limits the size of `Arrays.scala`, which would have been _very_large_ with the prior approach.
It also sets the stage and provides a model for  JDK > 8 additions in other files, when the 
additions are large (say, I think,  for `Buffers`.)